samples/helm-sample-application/application/src/cloud-business-partner/cloud-business-partner.service.ts

```diff
@@ -1,18 +1,19 @@
 import { Injectable } from '@nestjs/common';
 import {
   BusinessPartner,
-  businessPartnerService,
-} from '@sap/cloud-sdk-vdm-business-partner-service';
+  cloudBusinessPartnerService as businessPartnerService,
+} from '../generated/cloud-business-partner-service';
 const { businessPartnerApi } = businessPartnerService();
 
-const destinationName: string = process.env.CLOUD_DESTINATION;
```

Comment:
I believe this was accidentally removed while the code was copied between samples?
This is helm-specific logic, so this part has to be reverted.

Reply:
ups yes good catch.